### PR TITLE
add singlerider keys

### PR DESCRIPTION
**Pull request by singlerider**

### CODE REVIEW

File: group_vars/all/users.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ users:
   singlerider:
     irc_oper:
       type: sslclientcertfp
-      pass: "{{ vault_users.singlerider.irc_operpass }}"
+      pass: "41:AE:61:33:58:25:B2:BB:30:D1:07:1A:2C:F6:A7:CB:59:A6:E6:81:16:CA:86:82:09:2F:F3:F3:3F:0D:0E:88"
```
Comment (RyanSquared) [inline group_vars/all/users.yml]: shouldn't you have changed the value in `vault_users` instead?

Comment (singlerider) [reply group_vars/all/users.yml]: https://github.com/hashbang/admin-tools/blob/master/doc/New_admin.md#certificate-authentication

Comment (KellerFuchs) [reply group_vars/all/users.yml]: @RyanSquared No, that's valid, a certificate fingerprint is not a secret (unlike a password hash that can be bruteforced...)

**Review by KellerFuchs — CHANGES_REQUESTED** [review]
- As far as I can tell, the contents of the vault didn't change, so there is really no need to rekey it.
- Please squash the two commits, since the second one fixes the first.

**Review by KellerFuchs — APPROVED** [review]

**Pull request merged**